Updated the Install.MD to contain Hardware Requirements

knechtionscoding opened the pull request:
Added the hardware requirements of 2 CPUs and 4 GB of Ram. I took the lead from the Tower Documentation and assumed similar recommended hardware specs. Added the correct links in the TOC at the top of the page.

At the very least this adds a location for the minimum Hardware Recommendations. This information should also be added to the issues template to prevent troubleshooting on hardware limitations like #115.

knechtionscoding committed:
Updated the Install.MD to contain Hardware Requirements

Added the hardware requirements of 2 CPUs and 4 GB of Ram. I took the lead from the Tower Documentation and assumed similar recommended hardware specs. Added the correct links in the TOC at the top of the page.

matburt commented:
We're actually revising this upward for the 3.2 release of Tower, but I'm not sure how much I want to call out direct requirements in the docs. Let me think on this one. Will probably ask for revision.

knechtionscoding commented:
@matburt Sure. I just wanted to add the space and location for it now so that whatever requirements are decided on (even if they are scaling similar to the Tower reqs that measure it per fork) there is a place for them.

leifmadsen commented on INSTALL.md:
```diff
@@ -31,6 +34,15 @@ This document provides a guide for installing AWX.
+2 CPUs recommended.
+
+#### RAM
+2 GB RAM is the minimum .
```
Extra space prior to the period.

knechtionscoding replied:
@leifmadsen Why should there be an extra space prior to the period here but not at the " 4GB RAM is recommended." line?

leifmadsen replied:
@KnechtionsCoding that's exactly what I'm pointing out. It's inconsistent. The space prior to the period on this line should be removed.

knechtionscoding replied:
@leifmadsen Sorry, I clearly shouldn't comment after a long day. I'll fix that so it is consistent now.

knechtionscoding committed:
Updated to remove extra period and make consistent

Made the Hardware recommendation language consistent with is and are. Removed extra space.

wwitzel3 commented:
Added here. https://github.com/ansible/awx/commit/cf0cc2e2f2db8446c1f80538a0e504fb0a877efc
